doc: fix documented winmode argument default in ctypes CDLL classes. (#19167)
python/cpython: Doc/library/ctypes.rst

The *winmode* parameter is used on Windows to specify how the library is loaded
(since *mode* is ignored). It takes any value that is valid for the Win32 API
``LoadLibraryEx`` flags parameter. When it is ``None``, the default is to use

> the flags that result in the most secure DLL load to avoiding issues such as DLL

Reviewer: While you're here, replace "to avoiding" with "in order to avoid".

Author: now done

The *winmode* parameter is used on Windows to specify how the library is loaded
(since *mode* is ignored). It takes any value that is valid for the Win32 API

> ``LoadLibraryEx`` flags parameter. When it is ``None``, the default is to use
> the flags that result in the most secure DLL load in order to avoid issues such
> as DLL hijacking. Passing the full path to the DLL is the safest way to ensure
> the correct library and dependencies are loaded.

Reviewer: The documented default value of `winmode` has since been fixed as `None` instead of 0. The only remaining problem that I see is the phrase "to avoiding issues".

```suggestion
``LoadLibraryEx`` flags parameter. When omitted, the default is to use the
flags that result in the most secure DLL load, which avoids issues such as DLL
hijacking. Passing the full path to the DLL is the safest way to ensure the
correct library and dependencies are loaded.
```

I don't think `None` needs to be called out explicitly. The reader can see what the default value is in the signature of the constructor.
